js/app.js

```diff
@@ -1,234 +1,139 @@
-  },
-  seaCTime: function(){
-    var seattleCenter = document.getElementById('SeattleCenter');
-    var total = 0;
-    for(var i = 0; i < time.length; i++){
-      var seaCentEl = document.createElement('li');
-      var randomNum =seaC.seaCCookies();
-      seaCentEl.textContent = 'The Seattle Center sold ' + randomNum + ' cookies from '+ time[i] + '.';
-      total += randomNum;
-      seattleCenter.appendChild(seaCentEl);
-    }
-    var totalCookies = document.createElement('li');
-    totalCookies.textContent = 'Total number of cookies sold at the Seattle Center is ' + total;
-    seattleCenter.appendChild(totalCookies);
-  },
+City.prototype.total = function(){
+  var cookieAmount = 0;
+  for (var i=0; i < this.cookiesPerHour.length; i++ ){
+    cookieAmount += this.cookiesPerHour[i];
+  }
+  return this.totalCookies.push(cookieAmount);
 };
 
-var capHill={
-  minP: 20,
-  maxP: 38,
-  avgP: 2.3,
-  capHillCustomers: function(){
-    return Math.round(Math.random() * (capHill.maxP - capHill.minP)) + capHill.minP;
-  },
-  capHillCookies: function(){
-    return Math.round(capHill.capHillCustomers() * capHill.avgP);
-  },
-  capHillTime:function(){
-    var capitolHill = document.getElementById('CapitolHill');
-    var total = 0;
-    for(var i = 0; i < time.length; i++){
-      var capHillEl = document.createElement('li');
-      var randomNum =capHill.capHillCookies();
-      capHillEl.textContent = 'Capitol sold ' + randomNum + ' cookies from '+ time[i] + '.';
-      total += randomNum;
-      capitolHill.appendChild(capHillEl);
+
+var testFunction = function (){
+  for (var r=0; r < timeTable.length; r++) {
+
+    var hourlyTotal = 0;
+    for (var i=0; i < allCities.length; i++){
+
+      hourlyTotal += allCities[i].cookiesPerHour[r];
     }
-    var totalCookies = document.createElement('li');
-    totalCookies.textContent = 'Total number of cookies sold at Capitol Hill is ' + total;
-    capitolHill.appendChild(totalCookies);
+    totalAmountEachHour.push(hourlyTotal);
+    console.log('-------------');
   }
+  console.log(totalAmountEachHour);
+
 };
 
-var alki={
-  minP: 2,
-  maxP: 16,
-  avgP: 4.6,
-  alkiCustomers:function() {
-    return Math.round(Math.random() * (alki.maxP - alki.minP)) + alki.minP;
-  },
-  alkiCookies: function(){
-    return Math.round(alki.alkiCustomers() * alki.avgP);
-  },
-  alkiTime: function(){
-    var alkiAlki = document.getElementById('Alki');
-    var total = 0;
-    for(var i = 0; i < time.length; i++){
-      var alkiEl = document.createElement('li');
-      var randomNum =alki.alkiCookies();
-      alkiEl.textContent = 'Alki sold ' + randomNum + ' cookies from '+ time[i] + '.';
-      total += randomNum;
-      alkiAlki.appendChild(alkiEl);
-    }
-    var totalCookies = document.createElement('li');
-    totalCookies.textContent = 'Total number of cookies sold at Alki is ' + total;
-    alkiAlki.appendChild(totalCookies);
 
+
+// start of table stuff
```

Comment:
When creating your table data, you want to start with adding content to the smallest units of data and append to the parent last, then appending each row to the table, and so on.